packages/cloudvision-grpc-web/src/grpc/index.ts

```diff
@@ -30,6 +30,9 @@ const DEFAULT_CONTROL_FUNCTIONS = {
   },
 };
 
+const myTransport = grpc.CrossBrowserHttpTransport({ withCredentials: true });
```

Comment:
Let's not make this the default. The default can always be set by the user, or passed as an option to the invoke calls

Reply:
Will do